src/tests/all-tests.js

```diff
@@ -0,0 +1,272 @@
+/* Tests for the "express" and "router" module */
+"use strict";
+var namespace    = "gpii.express.tests";
+var fluid        = fluid || require('infusion');
+var gpii         = fluid.registerNamespace("gpii");
+var expressTests = fluid.registerNamespace(namespace);
+var path         = require("path");
+var jqUnit       = fluid.require("jqUnit");
+var request      = require("request");
+
+require("../js/express");
+require("../js/router");
+require("../js/static");
+require("../js/middleware");
+require("../js/bodyparser");
+require("../js/cookieparser");
+require("../js/session");
+require("./js/test-middleware");
+require("./js/test-router");
+require("./js/test-router-reqview");
+
+expressTests.isSaneResponse = function(jqUnit, error, response, body) {
```

Comment:
Rename this function to something descriptive. It appears to verify that we get 200 code with no error.